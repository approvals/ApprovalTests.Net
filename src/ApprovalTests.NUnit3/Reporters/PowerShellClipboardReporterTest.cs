[TestFixture]
public class PowerShellClipboardReporterTest
{
    [Test]
    public void TestCommand() =>
        Approvals.Verify(PowerShellClipboardReporter.GetCommandLineForApproval(@"c:\temp\approved.txt", @"c:\temp\recieved.txt"));
}