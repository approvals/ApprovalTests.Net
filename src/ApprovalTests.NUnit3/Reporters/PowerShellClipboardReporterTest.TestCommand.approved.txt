Move-Item "c:\temp\recieved.txt" "c:\temp\approved.txt" -Force